### PR TITLE
Updated translations

### PR DESCRIPTION
**Pull request by lee-delarm6**
Updated translations

## Summary

- Added translations for spanish and tagalog

## Testing done

- Made sure app still built, no translations in use

### CODE REVIEW

**Review by brianseek — CHANGES_REQUESTED** [review]
These should be alphabetized along with the corresponding english keys if not already alphabetized there.

**Comment by lee-delarm6** [comment]
> These should be alphabetized along with the corresponding english keys if not already alphabetized there.

Gotcha, alphabetizing also moved a couple others around, but I assume it's no worry

File: src/applications/check-in/locales/en/translation.json [hunk 1]
```diff
@@ -101,143 +123,121 @@
...
   "youre-traveling-in-relation-to-a-compensation-and-pension": "A scheduled VA claim exam (also called a compensation and pension, or C&P, exam), to get a service dog, or for VA-approved transplant care.",
-  "how-to-update-next-of-kin": "How to update your next of kin information",
-  "confirm-who-youd-like-to-represent-your-wishes": "Confirm who you'd like to represent your wishes for care, medical documentation, and benefits if needed. Your next of kin is often your closest living relative, like your spouse, child, parent, or sibling.",
-  "if-this-isnt-your-correct-information-a-staff-member-can-help--dayOf": "If this isn’t your correct information, select <0>No</0> and a staff member can help you check in and update your information.",
-  "if-this-isnt-your-correct-information-a-staff-member-can-help--preCheckIn": "If this isn’t your correct information, select <0>No</0> and a staff member can help you update your information on the day of your appointment."
+  "youve-completed-pre-check-in": "You’ve completed pre-check-in",
+  "zip-code-is-required": "Zip code is required"
```
[the remark this answers is not in the document]
Comment (lee-delarm6) [reply src/applications/check-in/locales/en/translation.json:242]: Oh shoot, I didn't take into account other new translations. Good catch!

**Review by brianseek — CHANGES_REQUESTED** [review]
One change to the english file and I think this will be good to go.

**Comment by lee-delarm6** [comment]
Re-added those translations to the bottom as requested @brianseek

**Review by brianseek — APPROVED** [review]
Looks good!

**Pull request merged**